elastic/kibana
# [TSVB] should call renderComplete once for `multi-value` visualizations

## Summary

This PR applies the same changes for `TSVB` as for `Agg-based` visualizations (https://github.com/elastic/kibana/pull/142842)


### How to reproduce 

1. Create TSVB Metric with grouping data by some field
<img width="1161" alt="image" src="https://user-images.githubusercontent.com/20072247/196681238-ce9cc64e-7d30-433d-a78e-5d2dd70ce068.png">

2. Open Browser Dev Tools and see how many times we executed `renderComplete`

Expected result: 
`hook should be executed once. `

### Screens

#### Before

<img width="1685" alt="Screenshot 2022-10-19 at 14 36 01" src="https://user-images.githubusercontent.com/20072247/196681511-bcc6d9b0-09d0-475d-bba7-12ec54e75a67.png">

#### After

<img width="1685" alt="Screenshot 2022-10-19 at 14 34 41" src="https://user-images.githubusercontent.com/20072247/196681545-64420b6d-e4da-4c7b-9c84-f93919ccbc4f.png">
